### ISSUE TITLE
Use astropy units when possible

### ISSUE THREAD

**Issue by morcuended** [JaimeRosado/ShowerModel#18]

**Comment by JaimeRosado** [comment]
I do not think that it is useful, because there is almost no unit conversions in the code and data is stored mostly in DataFrames that do not support astropy units. In addition, this would involve a new dependency and force the user to import astropy.units module to input data in different units.
